model_compression_toolkit/core/common/graph/base_node.py

```diff
@@ -363,3 +363,35 @@ def find_max_candidates_indices(self) -> List[int]:
                 max_candidates.append((i, c))
 
         return [i for i, a_n_bits in max_candidates]
+
+    def get_unique_weights_candidates(self) -> List[Any]:
+        """
+        Returns a list with node's candidates of unique weights bit-width value.
+        If the node have multiple candidates with the same weights bit-width, the candidate with the highest
+        activation bit-width is returned.
+
+        Returns: A list with node's candidates of unique weights bit-width value.
+        """
+
```

Comment:
I would remove the comment about the bit-width tie-breaker as it assumes the candidates are sorted (or maybe this assumption just need to be asserted).

Reply:
We count on this assumption widely li other places in the code as well, so I don't think an assertion here will make any change.
I'll change the comment.